scan/tls/alert.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
It might even make sense to leave this as its own repository (forked from crypto/tls) so that we can track the diffs and keep it up to date with Go's tls changes.